src/gui/qgsattributeeditorcontext.h

```diff
@@ -261,11 +264,24 @@ class GUI_EXPORT QgsAttributeEditorContext
       return metaEnum.valueToKey( static_cast<int>( mAttributeFormMode ) );
     }
 
+    /**
+     * Set current \a messageBar as the main information bar
+     * \since QGIS 3.12
+     */
+    void setMainInfoBar( QgsMessageBar *messageBar ) { mMainInfoBar = messageBar; }
```

Comment:
Is the term `mainInfoBar` (or just `infoBar`) used often? Or should `messageBar` be used which would be more coherent in taxonomy?

Reply:
Is `MainMessageBar` OK?

My point was to be clear about what messageBar it is . This is the "main" one, the "application" one, the "canvas" one. This is not the one that you have in some widget like the QgsAttributeForm for instance.

Reply:
I see. My point was indeed rather on the `Info` than on the `main` part of the name. Maybe canvas would be good (since it's used to for reporting messages for the map canvas tool), but no strong opinion.

Reply:
I choose mainMessageBar because it's not mandatory to be in the canvas (it is actually but could be otherwise)